code/modules/client/client procs.dm

```diff
@@ -906,3 +906,33 @@
 	return TRUE
 
 #undef SSD_WARNING_TIMER
+
+/client/verb/resend_ui_resources()
+	set name = "Reload UI Resources"
+	set desc = "Reload your UI assets if they are not working"
+	set category = "Special Verbs"
+
+	var/choice = alert(usr, "This will reload your NanoUI and TGUI resources. If you have any open UIs this may break them. Are you sure?", "Resource Reloading", "Yes", "No")
+	if(choice == "Yes")
+		// 600 deciseconds = 1 minute
+		if((last_ui_resource_send) < world.time)
+			last_ui_resource_send = world.time + 600
```

Comment:
`60 SECONDS`

Reply:
Im not used to these new-fangled defines